### PR TITLE
🙌🏾 Added support for sqlserver

### PR DESCRIPTION
**Pull request by sonylomo**
Closes issue #48

### CODE REVIEW

File: components/Models.tsx [hunk 1]
```diff
@@ -404,7 +404,7 @@ export default function Models() {
             New model
           </Button>
 
-          {schema.database !== "sqlite" && (
+          {schema.database !== "sqlite" && schema.database !== "sqlserver" && (
```
Comment (albingroen) [inline components/Models.tsx:407]: If we want to check for more providers in the future, maybe this syntax is a bit more maintainable.

```
!["sqlite", "sqlserver"].includes(schema.database)
```

File: pages/schemas/[schemaId]/models/[id]/index.tsx [hunk 2]
```diff
@@ -112,23 +124,14 @@ const Model = () => {
...
-                      if (schema.models.some((m: Model) => m.name === name)) {
-                        toast.error(`A model called ${name} already exists`);
-                        setName(model.name);
-                      } else {
-                        updateModel({ name });
-                      }
-                    }
-                    setEditingName(!editingName);
-                  }}
+                  onClick={()=> handleFieldEdit()}
```
Comment (albingroen) [inline pages/schemas/[schemaId]/models/[id]/index.tsx:134]: This can be
```
onClick={handleFieldEdit}
```

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "autoprefixer": "^10.4.0",
     "axios": "^0.24.0",
     "html-metadata-parser": "^2.0.4",
-    "next": "^12.0.7",
+    "next": "^12.2.4",
```
Comment (albingroen) [inline package.json:19]: Any specific reason for updating Next.js?

**Review by albingroen — CHANGES_REQUESTED** [review]

**Comment by albingroen** [comment]
Nice work, just had a few comments 👍🏼

File: package.json (hunk 3 above)
Comment (sonylomo) [reply package.json:19]: Not really. I didn't see any breaking changes that the package upgrade introduced. Would you prefer I roll it back to the previous version?

Comment (albingroen) [reply package.json:19]: Alright, no worries - just wanted to double check!

**Comment by albingroen** [comment]
Before I approve, do you mind squashing your commits?

**Review by albingroen — APPROVED** [review]

**Pull request merged**